김현수/mvvmstudy/app/src/main/java/com/example/mvvmstudy/UI/MainActivity.kt

```diff
@@ -0,0 +1,176 @@
+package com.example.mvvmstudy.UI
+
+import android.annotation.SuppressLint
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import retrofit2.Callback
+import android.view.Menu
+import android.view.MenuItem
+import android.widget.SearchView
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.mvvmstudy.R
+import com.example.mvvmstudy.IO.RetrofitClass
+import com.example.mvvmstudy.data.Repositories
+import com.example.mvvmstudy.databinding.ActivityMainBinding
+import org.jetbrains.anko.indeterminateProgressDialog
+import retrofit2.Call
+import retrofit2.Response
+
+class MainActivity : AppCompatActivity() {
+
+    private var page = 0
+    private var search =""
+
+    private var _binding : ActivityMainBinding?= null
```

Comment:
code format 하면 좋을 것 같아요!
```suggestion
    private var page = 0
    private var search = ""

    private var _binding : ActivityMainBinding? = null
```